### DISCUSSION TITLE
Meeting Sync

### DISCUSSION THREAD

**Discussion by Eric-Anderderson** [code4recovery/12-step-meeting-list#1265] [Q&A] [answered]
We received an email regarding data source changes detected-screenshot.1
![Email notification for removal of meetings](https://github.com/code4recovery/12-step-meeting-list/assets/150064201/7602ac54-4c9e-42a0-a096-9405cd543c7b)

It shows that the database only has 75 meetings. We checked our sheet and nothing has changed and we have 377 meetings listed. screenshot.2
![Meeting Count-377](https://github.com/code4recovery/12-step-meeting-list/assets/150064201/eb542863-fd5d-467b-81e9-97f509c34af1)

We logged into the Import/Export menu and the refresh began immediately without being prompted by click. When navigating to that menu it displayed 75 meetings as well. Which is inaccurate. And after the Refresh Bar completed, it now shows we have 401 meetings.

Here is the spreadsheet we are using: [https://docs.google.com/spreadsheets/d/1otk733FYwzF7JA4NlXfgEy6E1JKAkWOAxabjSOuuDIE/edit?usp=sharing](url)

This is the second time we got this email. The first time I just canceled the feed and reimported bc I thought it was human error. But this now the 2nd time seems to be a bigger issue.

Please advise on what may be happening and how we can address the issue?

**Comment by joshreisner** [comment, marked as answer]
hm, we'll look into it! sounds like change detection may not work as expected when importing from a google sheet

for now probably just ignore these emails

**Reply by SIA-WEBSUP** [reply]
Trying to reproduce on a clean install...

Have never tried to use a google sheet as source... 

I assume that we need to run through https://sheets.code4recovery.org/ to convert to a JSON?

When I run 'https://docs.google.com/spreadsheets/d/1otk733FYwzF7JA4NlXfgEy6E1JKAkWOAxabjSOuuDIE/' through, I get the following... is this normal??
![image](https://github.com/code4recovery/12-step-meeting-list/assets/63924390/cacb039d-d9ee-4252-9c13-8dc56e1fbd85)

JSON is empty...
https://sheets.code4recovery.org/storage/1otk733FYwzF7JA4NlXfgEy6E1JKAkWOAxabjSOuuDIE.json

**Reply by joshreisner** [reply]
@SIA-WEBSUP they are not using the sheets project, they have added the google sheet to TSML directly. please see this issue for reference: https://github.com/code4recovery/sheets/issues/23
